netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -211,4 +213,15 @@ protected NettyServer buildTransportServer() {
             workerEventLoopGroup, sslContext, maxConcurrentCallsPerConnection, flowControlWindow,
             maxMessageSize);
   }
+
+  @Override
+  public NettyServerBuilder useTransportSecurity(File certChain, File privateKey) {
+    try {
+      sslContext = GrpcSslContexts.forServer(certChain, privateKey).build();
+    } catch (SSLException e) {
+      // This should likely be some other, easier to catch exception.
```

Comment:
I would not think so.  Typically, builders throw on .build(), allowing invalid states to temporarily exist during construction.